src/Component.php

```diff
@@ -241,6 +243,35 @@ private function handleFor( DOMNode $node, array $data ) {
 		}
 	}
 
+	private function appendHTML( DOMNode $parent, $source ) {
+		$tmpDoc = new DOMDocument();
+		$tmpDoc->loadHTML( $source );
+		foreach ( $tmpDoc->getElementsByTagName('body')->item(0)->childNodes as $node ) {
+			$node = $parent->ownerDocument->importNode( $node, true );
+			$parent->appendChild( $node );
+		}
+	}
+
+	private function handleRawHtml( DOMNode $node, array $data ) {
+		if ( $this->isTextNode( $node ) ) {
+			return;
+		}
+
+		/** @var DOMElement $node */
+		if ( $node->hasAttribute( 'v-html' ) ) {
+			$variableName = $node->getAttribute( 'v-html' );
+			$node->removeAttribute( 'v-html' );
+
+			$newNode = $node->cloneNode( true );
+
+			$this->appendHTML( $newNode, $data[$variableName] );
+
+			$node->parentNode->insertBefore( $newNode, $node );
+
+			$this->removeNode( $node );
```

Comment:
This looks like it should be `$node->parentNode->replaceChild( $newNode, $node )`